### ISSUE TITLE
[Bug] Wrong Rotation Plugin used

### ISSUE THREAD

**Issue by StarkITA** [ffxivcode/AutoDuty#698]
### What happened?

Despite RSR being selected, when Wrath is installed AutoDuty enables Wrath and not RSR.

Wrath has no modules enabled on the class I was in, it just existed.

### Version Number

0.0.0.179

### Steps to reproduce the error

enter in duty with both wrath and RSR enabled.

**Comment by erdelf** [comment]
correct. Wrath was made for automation, RSR was not. Temporarily wrath is just prioritized. Waiting on the IPC on their side so I can properly detect the config made in wrath.

**Comment by StarkITA** [comment]
Thanks!
